test/functional/rpc_createmultisig.py

```
@@ -158,8 +158,8 @@ def do_multisig(self):
             assert mredeemw == mredeem
             wmulti.unloadwallet()
 
-        spk = bytes.fromhex(node0.validateaddress(madd)["scriptPubKey"])
-        txid, _ = self.wallet.send_to(from_node=self.nodes[0], scriptPubKey=spk, amount=2000)
+        spk = address_to_scriptpubkey(madd)
```

Ruff (0.12.2)

Undefined name `address_to_scriptpubkey`

(F821)